_data/simple-icons.json

```
@@ -4352,8 +4352,9 @@
         },
         {
             "title": "Fujifilm",
-            "hex": "ED1A3A",
-            "source": "https://upload.wikimedia.org/wikipedia/commons/a/a1/Fujifilm_logo.svg"
+            "hex": "FB0020",
+            "source": "https://www.fujifilm.com/us/themes/custom/fujifilm_com_g2/common/img/fujifilm_corporate_logo.svg",
```

```suggestion
            "source": "https://www.fujifilm.com",
```
I'd suggest we just link to the website, if it's available in the header - saves us getting a 404 if/when they change the filename! 🙃